photo.go

```diff
@@ -65,6 +94,70 @@ func (p *ImgixPhoto) GetThumbnailForWidthAndHeight(w, h int) string {
 	return fullUrl.String()
 }
 
+func (p *ThumborRaw) GetPhotoForWidth(w int) string {
+	thumborOptions := gothumbor.ThumborOptions{Width: w, Smart: true}
+	thumborPath, err := gothumbor.GetCryptedThumborPath(p.Secret, p.Key, thumborOptions)
+	if err != nil {
+		fmt.Print(err)
+	}
+
+	parsedPath, err := url.Parse(thumborPath)
+	fullUrl := p.BaseUrl.ResolveReference(parsedPath)
+
+	return fullUrl.String()
+}
+
+func (p *ThumborRaw) GetThumbnailForWidthAndHeight(w, h int) string {
+	thumborOptions := gothumbor.ThumborOptions{Width: w, Height: h, Smart: true}
+	thumborPath, err := gothumbor.GetCryptedThumborPath(p.Secret, p.Key, thumborOptions)
+	if err != nil {
+		fmt.Print(err)
+	}
+
+	parsedPath, err := url.Parse(thumborPath)
+	fullUrl := p.BaseUrl.ResolveReference(parsedPath)
+
+	return fullUrl.String()
+}
+
+func (p *ThumborCloudfront) SignCloudfrontURL(path string) string {
+
+	parsedPath, err := url.Parse(path)
+	if err != nil {
+		log.Fatalf("Failed to parse URL for signing, err: %s\n", err.Error())
```

Comment:
To keep in line with the other functions that return paths, it might be better to use `log.Error` and `return ""` instead of killing the server.